projects/akita-ng-fire/src/lib/factory/factory.service.ts

```diff
@@ -30,17 +51,55 @@ export class FactoryService<S extends EntityState<any, string>> {
     return this.constructor['idKey'] || 'id';
   }
 
+  /** An observable version of the path */
+  get path$(): Observable<string> {
+    return of(this.path);
+  }
+
   /** Preformat the document before updating Firestore */
   protected preFormat<E extends getEntityType<S>>(document: Readonly<Partial<E>>): E {
     return document;
   }
 
-  syncStore(name: string, queryFn?: QueryFn) {
-    return this.db
-      .collection<getEntityType<S>>(this.path, queryFn)
-      .stateChanges().pipe(
-        withTransaction(actions => syncStoreFromAction(name, actions))
-      );
+  /** Stay in sync with the collection or a fractio of it */
+  syncCollection(name: string, path?: string | Observable<string> | QueryFn): Observable<DocumentChangeAction<getEntityType<S>>[]>;
```

Comment:
Change "name" to "storeName" for better understanding.

Reply:
6949b37